Upstream: Adapted from https://github.com/psf/requests/pull/5651
Reason: Fixes warning about urllib3 being outdated

diff --git a/requests/__init__.py b/requests/__init__.py
index db090aa..04ab8ea 100644
--- a/requests/__init__.py
+++ b/requests/__init__.py
@@ -60,7 +60,7 @@ def check_compatibility(urllib3_version, chardet_version):
     # urllib3 >= 1.21.1, <= 1.25
     assert major == 1
     assert minor >= 21
-    assert minor <= 25
+    assert minor <= 26
 
     # Check chardet for compatibility.
     major, minor, patch = chardet_version.split('.')[:3]
diff --git a/setup.py b/setup.py
index 5147636..cca8b94 100755
--- a/setup.py
+++ b/setup.py
@@ -44,7 +44,7 @@ packages = ['requests']
 requires = [
     'chardet>=3.0.2,<4',
     'idna>=2.5,<3',
-    'urllib3>=1.21.1,<1.26,!=1.25.0,!=1.25.1'
+    'urllib3>=1.21.1,<1.27'
 
 ]
 test_requirements = [
